src/validation.cpp

```diff
@@ -498,6 +500,7 @@ class MemPoolAccept
                             /* m_test_accept */ test_accept,
                             /* m_allow_bip125_replacement */ true,
                             /* m_package_submission */ false,
+                            /* m_allow_witness_replace */ true,
```

Comment:
```suggestion
                            /*m_allow_witness_replace=*/true,
```
Anywhere you are adding / touching named args, please use the [style from the developer docs](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c-named-arguments).